grouper/fe/handlers/group_edit_member.py

```diff
@@ -11,22 +14,30 @@
 from grouper.plugin.exceptions import PluginRejectedGroupMembershipUpdate
 from grouper.user import user_role
 
+if TYPE_CHECKING:
+    from typing import Any, Optional
+
 
 class GroupEditMember(GrouperHandler):
-    def get(self, group_id=None, name=None, name2=None, member_type=None):
-        group = Group.get(self.session, group_id, name)
+    def get(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Out of curiosity, why use `args`/`kwargs` here?

Reply:
Sad Tornado nonsense. The Tornado typeshed declares the type of these methods as this generic type, so you either have to add type ignore comments or you have to live with it and pull things out of `kwargs` with explicit typing. I really hate type ignore comments, so decided to live with this. The same is done in all the other Tornado handlers that have been modified since we started doing mypy.